README.md

```diff
@@ -1,66 +1,231 @@
-# terraform-provider-oneview
+# HPE OneView SDK for Terraform Provider
 
-[![Build Status](https://travis-ci.org/HewlettPackard/terraform-provider-oneview.svg?branch=master)](https://travis-ci.org/HewlettPackard/terraform-provider-oneview)
+## Build Status 
 
-A Terraform provider for oneview
+OV Version | 5.60   | 5.50   | 5.40    
+| ------------- |:-------------:| -------------:| -------------:|
+SDK Version/Tag | [v1.7.0-12 ](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.7.0-12) | [v1.7.0-12 ](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.7.0-12) |[v1.7.0-12 ](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.7.0-12) |
```

Comment:
all the 3 tag names are same. Why so..?

Reply:
We do not have older releases for terraform 12

Reply:
then keep only one release.. remove the remaining 2 rows